extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQSpec.java

```diff
@@ -43,7 +43,7 @@ public class MSQSpec
   @JsonCreator
   public MSQSpec(
       @JsonProperty("query") Query<?> query,
-      @JsonProperty("columnMappings") @Nullable ColumnMappings columnMappings,
+      @JsonProperty("columnMappings") ColumnMappings columnMappings,
```

Comment:
Is there a reason null-able is removed ?

Reply:
There is a not-null check below.